src/__tests__/createAction-test.js

```diff
@@ -95,7 +95,8 @@ describe('createAction()', () => {
 
       const explictNullAction = createAction(type)(null);
       expect(explictNullAction).to.deep.equal({
-        type
+        type,
+        payload: null
       });
 
       const baz = '1';
```

Comment:
Can we get rid of this test? It doesn't have anything to do with the test name and is already being tested [here](https://github.com/acdlite/redux-actions/pull/128/files#diff-cd3494025bc48661a763dcb654f5a116R91).